src/main/resources/application.yaml

```diff
@@ -81,6 +81,7 @@ service-config:
     - service: divorce
       jurisdiction: DIVORCE
       case-type-ids: []
+      transformation-url: ${TRANSFORMATION_URL_DIVORCE}
```

Comment:
you would need some default value in `values.yaml` else app won't boot up in prod and hat prod containers ?

Reply:
I moved `transformation-url` to `values.yaml` file. I'll raise a pr to update flux repo to set the `transformation-url` value to an empty string in prod.
https://github.com/hmcts/cnp-flux-config/blob/c788e0cdc59d7fb332303a825beb3e6233adea32/k8s/prod/common/bsp/bulk-scan-orchestrator.yaml#L24

Reply:
https://github.com/hmcts/cnp-flux-config/pull/1643